app/scripts/directives/unbindService.js

```diff
@@ -0,0 +1,121 @@
+    //     deploymentConfigs = _.toArray(deploymentConfigData.by('metadata.name'));
+    //     ctrl.applications = sortApplications(deploymentConfigs, deployments, replicationControllers, replicaSets, statefulSets);
+    //   });
+    //   DataService.list('replicationcontrollers', context).then(function(replicationControllerData) {
+    //     replicationControllers = _.reject(replicationControllerData.by('metadata.name'), $filter('hasDeploymentConfig'));
+    //     ctrl.applications = sortApplications(deploymentConfigs, deployments, replicationControllers, replicaSets, statefulSets);
+    //   });
+    //   DataService.list({
+    //     group: 'extensions',
+    //     resource: 'deployments'
+    //   }, context).then(function(deploymentData) {
+    //     deployments = _.toArray(deploymentData.by('metadata.name'));
+    //     ctrl.applications = sortApplications(deploymentConfigs, deployments, replicationControllers, replicaSets, statefulSets);
+    //   });
+    //   DataService.list({
+    //     group: 'extensions',
+    //     resource: 'replicasets'
+    //   }, context).then(function(replicaSetData) {
+    //     replicaSets = _.reject(replicaSetData.by('metadata.name'), $filter('hasDeployment'));
+    //     ctrl.applications = sortApplications(deploymentConfigs, deployments, replicationControllers, replicaSets, statefulSets);
+    //   });
+    //   DataService.list({
+    //     group: 'apps',
+    //     resource: 'statefulsets'
+    //   }, context).then(function(statefulSetData) {
+    //     statefulSets = _.toArray(statefulSetData.by('metadata.name'));
+    //     ctrl.applications = sortApplications(deploymentConfigs, deployments, replicationControllers, replicaSets, statefulSets);
+    //   });
+    // };
+
+    var unbindService = function() {
+      DataService.delete({
+        group: 'servicecatalog.k8s.io',
+        resource: 'bindings'
+      }, ctrl.selectedBinding, context).then(_.noop, function(err) {
+        ctrl.error = err;
+      });
+    };
+
+    var showService = function() {
+      ctrl.nextTitle = 'Delete';
+    };
+
+    var showConfirm = function() {
+      ctrl.nextTitle = 'Close';
+      ctrl.wizardComplete = true;
+      unbindService();
+    };
+
+    ctrl.$onInit = function() {
+      // loadApps();
+      ctrl.displayName = getDisplayName(ctrl.target);
+      ctrl.steps = [{
+        id: 'service',
+        label: 'Service',
+        view: 'views/directives/bind-service/select-binding.html',
+        onShow: showService
+      }, {
+        id: 'confirmation',
+        label: 'Confirmation',
+        view: 'views/directives/bind-service/remove-binding-confirm.html',
+        onShow: showConfirm
+      }];
+
+      context = {
+        namespace: _.get(ctrl.target, 'metadata.namespace')
+      };
+    };
+
+    ctrl.appNameForBinding = function(bindingName) {
+      return _.get(_.first(_.get(ctrl.applications, bindingName)), ['metadata', 'name']) || bindingName;
```

Comment:
Updating to this:

![screen shot 2017-06-14 at 1 19 32 pm](https://user-images.githubusercontent.com/280512/27145532-2d803bb0-5104-11e7-90ab-417cdf200aa2.png)

@ncameronbritt for input if this deviates or is new.  In theory, if a binding represents multiple apps, this is a bit wasteful of space as they could be listed horizontally, but I don't know of another pattern off the top of my head so going with consistency for now.  We can iterate if needed.